ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-icon.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
can we reuse form teleop server?

Reply:
No. The icons are very different, different folders, different launch scripts. There are some similarities and some overlaps sure, but it was much quicker, easier and less risky to just focus on server_anc_nuc icons without impacting teleop_icons and vice versa. Refactoring this to minimize repetition and somehow combine parts of server_and_nuc and teleop icons is probably difficult, would cause non-intuitive dependencies and would introduce new bugs/issues we haven't thought about.